src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordCopyCtor.cs

```diff
@@ -37,7 +37,6 @@ internal sealed class SynthesizedRecordCopyCtor : SynthesizedInstanceConstructor
 
         internal override void GenerateMethodBodyStatements(SyntheticBoundNodeFactory F, ArrayBuilder<BoundStatement> statements, BindingDiagnosticBag diagnostics)
         {
-            // Tracking issue for copy constructor in inheritance scenario: https://github.com/dotnet/roslyn/issues/44902
```

Comment:
What was this issue tracking, and has that been resolved?

Reply:
The issue was tracking how the copy constructor should work (should delegate to base copy ctor, except for object, and should not include initializers). The issue was closed as completed: https://github.com/dotnet/roslyn/issues/44902